packages/cli/test/unit/util/input/select-org.test.ts

```diff
@@ -4,33 +4,75 @@ import { useTeams } from '../../../mocks/team';
 import { useUser } from '../../../mocks/user';
 
 describe('selectOrg', () => {
+  let user;
+  let team;
+
+  beforeEach(() => {
+    team = useTeams()[0];
+  });
+
   describe('non-northstar', () => {
+    beforeEach(() => {
+      user = useUser();
+    });
+
     it('should allow selecting user', async () => {
-      const user = useUser();
-      useTeams();
       const selectOrgPromise = selectOrg(client, 'Select the scope');
       await expect(client.stderr).toOutput(user.name);
       client.stdin.write('\r'); // Return key
       await expect(selectOrgPromise).resolves.toHaveProperty('id', user.id);
     });
 
     it('should allow selecting team', async () => {
-      useUser();
-      const team = useTeams()[0];
       const selectOrgPromise = selectOrg(client, 'Select the scope');
       await expect(client.stderr).toOutput('Select the scope');
       client.stdin.write('\x1B[B'); // Down arrow
       client.stdin.write('\r'); // Return key
       await expect(selectOrgPromise).resolves.toHaveProperty('id', team.id);
     });
+
+    describe('with a selected team scope', () => {
+      beforeEach(() => {
+        client.config.currentTeam = team.id;
+      });
+
+      afterEach(() => {
+        delete client.config.currentTeam;
+      });
+
+      it('should allow selecting user', async () => {
+        const selectOrgPromise = selectOrg(client, 'Select the scope');
+        await expect(client.stderr).toOutput(user.name);
+        client.stdin.write('\r'); // Return key
+        await expect(selectOrgPromise).resolves.toHaveProperty('id', team.id);
+      });
+
+      it('should allow selecting team', async () => {
+        const selectOrgPromise = selectOrg(client, 'Select the scope');
+        await expect(client.stderr).toOutput('Select the scope');
+        client.stdin.write('\x1B[B'); // Down arrow
+        client.stdin.write('\r'); // Return key
+        await expect(selectOrgPromise).resolves.toHaveProperty('id', user.id);
+      });
```

Comment:
Are these the exact same tests as the ones above? You should use one of the test iterator methods instead so we don't have duplicate test code.

Reply:
I'm always -1 on test iterators because they make it harder to isolate a test when it fails and during development.

Reply:
I prefer duplication in test code to some degree. I think this is better without an iterator.